packages/speed.txt

```diff
@@ -0,0 +1,4 @@
+orjson>=3.5.4
+Brotli>=1.0.9
+aiodns>=1.1
+cchardet>=2.1.7
```

Comment:
How will these packages speed up Pincer? They need to be actually used in the code to have an effect, correct?